soccer/gameplay/generate_fsm_diagrams.py

```diff
@@ -21,8 +20,20 @@ def mkdir_p(path):
 logging.getLogger().setLevel(logging.INFO)
 
 import main
+import ui.main
 main.init()
 
+class moc_Point():
```

Comment:
Oh, I think I get it now, that lets python import the cpp bindings from the `robocup.so` file in run? That actually makes a lot of sense now that I think of it (so python isn't started specially from cpp at all, but just loads that so file). 

Adding:
```
sys.path.append('../../run')
import robocup
```
Works fine! @Jason27chan You can probably remove all these moc classes and replace them with real classes, as suggested above.